### PR TITLE
fix(video): consume errors for playing video

### PR DESCRIPTION
**Pull request by onx2**
Sometimes browsers disallow auto-play and when this happens we should consume the error. For now, this PR consumes all errors because each browser implementation is different and we are hitting our sentry limit because of this.

### CODE REVIEW

File: weave-js/src/components/Panel2/VideoViewer.tsx [hunk 1]
```diff
@@ -88,11 +88,10 @@ const VideoViewer = (props: VideoViewerProps) => {
     if (videoRef.current) {
       videoRef.current.muted = muted ?? true;
       if (autoPlay && videoRef.current.paused) {
-        try {
-          videoRef.current.play();
-        } catch (err) {
-          // sometimes we can't because chrome disallows unmuted autoplay. see: https://goo.gl/xX8pDD
-        }
+        videoRef.current.play().catch(() => {
+          // Browsers have their own implementations of how to handle auto-play and m
```
Comment (jamie-rasmussen) [inline weave-js/src/components/Panel2/VideoViewer.tsx:92]: You have an extra trailing 'm' in both of the comments

File: weave-js/src/components/Panel2/VideoViewer.tsx [hunk 2]
```diff
@@ -205,8 +204,14 @@ const VideoViewer = (props: VideoViewerProps) => {
                 onClick={e => {
                   const v = e.target as HTMLVideoElement;
                   if (v.paused) {
-                    v.play();
-                    setPlayButtonVisibility(false);
+                    v.play()
+                      .catch(() => {
+                        // Browsers have their own implementations of how to handle auto-play and m
```
Comment (onx2) [inline weave-js/src/components/Panel2/VideoViewer.tsx:209]:
```suggestion
                        // Browsers have their own implementations of how to handle auto-play and
```

File: weave-js/src/components/Panel2/VideoViewer.tsx (hunk 1 above)
Comment (onx2) [inline weave-js/src/components/Panel2/VideoViewer.tsx:92]:
```suggestion
          // Browsers have their own implementations of how to handle auto-play and
```

**Comment by onx2** [comment]
> So the thing here is that anything falling into try/catch hits Sentry but anything inside the `.catch()` handler is swallowed?

They aren't async functions so IIRC when I tested this, those aren't consumed unless we await which we can't in the current implementation. However, catch should work because it doesn't require async fn syntax

**Pull request merged**